extensions/interactions/TextInput/protractor.js

```diff
@@ -53,6 +53,7 @@ var submitAnswer = function(elem, answer) {
       inputElem.isPresent().then(function(present) {
         if (present) {
           inputElem.sendKeys(answer + '\n');
+          textInputElem.element(by.tagName('button')).click();
```

Comment:
Ditto. I am concerned about removing the ability to submit by pressing Enter. That's a regression. Can we re-incorporate that somehow?